sound/aloop/data.py

```diff
@@ -10,7 +10,7 @@
   for i in range(0, frames):
     s = i & 65535
     frame = [(s >> 8) & 255, s & 255, ((s >> 8) & 255) ^ 255, (s & 255) ^ 255]
-    fp.write(bytes(frame))
+    fp.write(bytearray(frame))
```

Comment:
@jkysela-rh thank you, will this fix it for both python2 and python3 ?

Reply:
Yes, I verified the fix with both python2 and python3.

Reply:
Thanks just checking :-) I'll merge and we can re-trigger the testing in kpet CI.